portfolio/src/main/webapp/script.js

```diff
@@ -27,3 +27,8 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+  async function getHello() {
```

Comment:
Fix indentation for this function.

function definition, and close of function should be aligned with functions above, content inside should be indented inside that.

Also please add a comment before getHello which quickly says what it is doing. (eg. Fetches Hello content from servlet, and sets it on the page).